### DISCUSSION TITLE
Having trouble in merge the image to matplotlib.

### DISCUSSION THREAD

**Discussion by Photonico** [materialsproject/pymatgen#2958] [Q&A]
My code works well 
```python
import matplotlib.pyplot as plt
from pymatgen.electronic_structure.plotter import BSPlotter
from pymatgen.io.vasp import Vasprun

# Extract the band structure data from the vasprun.xml file
vasprun = Vasprun("Bulk_fcc_Si_bandstructure/vasprun.xml")
band_structure = vasprun.get_band_structure()

# Use pymatgen's BSPlotter tool to create the band structure plot
plotter = BSPlotter(band_structure)
band_plot = plotter.get_plot(ylim=(-20, 40))
```

But I failed to merge the plotting into Matplotlib. Because I want to use Matplotlib to adjust the figure size, tick,  font size, font, etc. My other photo has the same style but this figure. I want to unify all the figures in my article.

For example. In my note: https://github.com/Photonico/OCM_Practice/blob/main/VASP/Analysis%20for%20Bulk%20Si.ipynb
I want to make the last picture becomes unified with the former pictures. 

How can I do this?

**Comment by janosh** [comment]
This seems more like a support question than an issue with `pymatgen`. Such questions are better asked on https://matsci.org/c/pymatgen or in GH discussion. I'll convert this to a discussion.
